packages/elements/src/interactive-chart/index.ts

```diff
@@ -40,7 +40,9 @@ import type {
   Theme,
   RowLegend,
   SeriesList,
+  SeriesData,
   SeriesDataItem,
+  SeriesOptions,
   SeriesStyleOptions,
   ColorToStringFunction
 } from './helpers/types';
```

Comment:
Let's align with other imports.
```suggestion
} from './helpers/types.js';
```